tardis/workflows/workflow_logging.py

```diff
@@ -0,0 +1,91 @@
+import logging
+
+import numpy as np
+import pandas as pd
+from IPython.display import display
+
+from tardis.io.logger.logger import logging_state
+from tardis.util.base import is_notebook
+
+logger = logging.getLogger(__name__)
+
+
+class WorkflowLogging:
+    def __init__(
+        self,
+        configuration,
+        log_level=None,
+        specific_log_level=None,
+    ):
+        logging_state(log_level, configuration, specific_log_level)
+
+    def log_plasma_state(
+        self,
+        t_rad,
+        dilution_factor,
+        t_inner,
+        next_t_rad,
+        next_dilution_factor,
+        next_t_inner,
+        log_sampling=5,
+    ):
+        """
+        Logging the change of the plasma state
+
+        Parameters
+        ----------
+        t_rad : astropy.units.Quanity
+            current t_rad
+        dilution_factor : np.ndarray
+            current dilution_factor
+        next_t_rad : astropy.units.Quanity
+            next t_rad
+        next_dilution_factor : np.ndarray
+            next dilution_factor
+        log_sampling : int
+            the n-th shells to be plotted
+
+        Returns
+        -------
+        """
+        plasma_state_log = pd.DataFrame(
+            index=np.arange(len(t_rad)),
+            columns=["t_rad", "next_t_rad", "w", "next_w"],
+        )
+        plasma_state_log["t_rad"] = t_rad
+        plasma_state_log["next_t_rad"] = next_t_rad
+        plasma_state_log["w"] = dilution_factor
+        plasma_state_log["next_w"] = next_dilution_factor
+        plasma_state_log.columns.name = "Shell No."
+
+        if is_notebook():
+            logger.info("\n\tPlasma stratification:")
```

Comment:
I think this should be a method called something like `log_notebook` to make it a little cleaner